libevm/VMSIMD.cpp

```diff
@@ -41,28 +41,30 @@ inline a32x8  const& v32x8 (u256 const& _stack_item) { return (a32x8&) *(a32x8*)
 inline a16x16 const& v16x16(u256 const& _stack_item) { return (a16x16&)*(a16x16*)&_stack_item; }
 inline a8x32  const& v8x32 (u256 const& _stack_item) { return (a8x32&) *(a8x32*) &_stack_item; }
 
+class Pow2Bits enum { BITS_8, BITS_16, BITS_32, BITS_64 };
```

Comment:
Wait, this doesn't compile for me at all. Did you intend it to be `enum class`? Then `BITS_8` etc. will be scoped (should be `Pow2Bits::BITS_8` etc. everywhere) and implicit conversion between `uint8_t` and `Pow2Bits` won't work in switches.
Probably easier to use plain enum, although coding standards discourage it